integration-tests/pom.xml

```diff
@@ -0,0 +1,212 @@
+<!-- This is unreleased proprietary source code of Oracle Corporation -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>oracle.kubernetes</groupId>
+    <artifactId>operator-parent</artifactId>
+    <version>1.0</version>
+  </parent>
+
+    <groupId>oracle.kubernetes</groupId>
+    <artifactId>operator-integration-tests</artifactId>
+    <version>1.0</version>
+
+    <description>Oracle Weblogic Server Kubernetes Operator</description>
+    <name>operator-integration-tests</name>
+    <packaging>jar</packaging>
+
+    <url>https://oracle.github.io/weblogic-kubernetes-operator</url>
+    <inceptionYear>2017</inceptionYear>
+    <licenses>
+      <license>
+        <name>The Universal Permissive License (UPL), Version 1.0</name>
+        <url>https://github.com/oracle/weblogic-kubernetes-operator/blob/master/LICENSE</url>
+      </license>
+    </licenses>
+	<dependencies>
+       <!-- test dependencies -->
+        <dependency>
+            <!-- note: need to include this before junit since junit includes an obsolete version of hamcrest -->
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-junit</artifactId>
+            <version>2.0.0.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit-version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-client</artifactId>
+            <version>${jersey-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-json-processing</artifactId>
+            <version>${jersey-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+            <version>${jersey-version}</version>
+        </dependency>
+    </dependencies>
+
+
+    <profiles>
+
+        <profile>
+            <id>java-integration-tests</id>
+            <properties>
+                <src-integration-test>${project.basedir}/src/integration-tests/java</src-integration-test>
+                <resource-integration-test>${project.basedir}/src/integration-tests/resources</resource-integration-test>
+                <docker-image-name>container-registry.oracle.com/middleware/weblogic-kubernetes-operator:latest</docker-image-name>
+                <skip.unit.tests>true</skip.unit.tests>
+            </properties>
+            <build>
+                <plugins>
+                	  <plugin>
```

Comment:
Please add the code format plugin:

			<plugin>
				<groupId>com.coveo</groupId>
				<artifactId>fmt-maven-plugin</artifactId>
				<version>2.4.0</version>
				<executions>
					<execution>
						<phase>test</phase>
						<goals>
							<goal>check</goal>
						</goals>
					</execution>
				</executions>
			</plugin>

Reply:
done